### ISSUE TITLE
Make it clear when fzy is not installed

### ISSUE THREAD

**Issue by jwoos** [cloudhead/neovim-fuzzy#5]
I installed fzy on my desktop and moved over the init.vim file to my laptop but I could not figure out why it would not load the plugin. I checked vim-plug, my neovim installation and reinstalled my plugins several times. I decided to look at the plugin and I saw 

```viml
if exists("g:loaded_fuzzy") || &cp || !executable('fzy') || !has('nvim')
  finish
endif
```

Perhaps a warning would be better? Silently not registering the plugin isn't very good UX

**Comment by cloudhead** [comment]
Good idea.

**Comment by digitalcosmos** [comment]
I'd like to add to this issue that [ag](https://github.com/ggreer/the_silver_searcher) should be mentioned as a requirement in README. Thanks!

**Comment by ysolis** [comment]
uhm, ag is an alias for apt-get in oh-my-zsh when ubuntu or debian plugin are enabled, will exist any posible problem if i call vim from a zsh shell with this alias enabled ?

**Comment by cloudhead** [comment]
The plugin shouldn't use your alias @ysolis, as it uses the vim `system()` function.

**Comment by cloudhead** [comment]
Fixed in 8b2f8b280f1bb52b8a33ceebf4177a4af2bf2735

It now gives you an error when you try to use it.
